packages/experimental/src/components/SidePanel/SidePanel.js

```diff
@@ -320,7 +321,9 @@ export const SidePanel = !pkg.isComponentEnabled(componentName)
           <>
             <div
               id={`${blockClass}-outer`}
-              className={mainPanelClassNames}
+              className={cx(mainPanelClassNames, {
+                [className]: className,
+              })}
```

Comment:
I think it will be neater to do this

```
className={cx(mainPanelClassNames, className)}
```

Also how it saw it in other components